operator/v1/types_network.go

```diff
@@ -256,6 +256,40 @@ type KuryrConfig struct {
 	// size by 1.
 	// +optional
 	OpenStackServiceNetwork string `json:"openStackServiceNetwork,omitempty"`
+
+	// enableDebug set to true will enable debug logs in kuryr-kuberntes.
+	// +optional
+	EnableDebug *bool `json:"enableDebug,omitempty"`
```

Comment:
I expected to see something more like https://github.com/openshift/api/blob/master/operator/v1/types_openshiftapiserver.go#L18 leading to https://github.com/openshift/api/blob/master/operator/v1/types.go#L60

Reply:
Nice, I can totally do logLevel, but in our case we don't have anything more verbose than debug. I'll explain that in the description.